pypyr/dsl.py

```diff
@@ -269,7 +270,9 @@ def _init_from_dict(self, step):
         # description: optional. Write to stdout if exists and flagged.
         self.description = step.get('description', None)
         if self.description:
-            logger.info("%s: %s", self.name, self.description)
+            logger.notice("%s: %s", self.name, self.description)
+        else:
+            logger.notice(self.name)
```

Comment:
change the else branch to `logger.debug("step name: %s:", self.name)`

Reply:
done